
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Not Acceptable
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.078+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "GET_Getallorganisationsthattheclientisauthorisedtoretrieve_406", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class GETGetallorganisationsthattheclientisauthorisedtoretrieve406 extends Exception {

    private java.lang.Object getGetallorganisationsthattheclientisauthorisedtoretrieve406;

    public GETGetallorganisationsthattheclientisauthorisedtoretrieve406() {
        super();
    }

    public GETGetallorganisationsthattheclientisauthorisedtoretrieve406(String message) {
        super(message);
    }

    public GETGetallorganisationsthattheclientisauthorisedtoretrieve406(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public GETGetallorganisationsthattheclientisauthorisedtoretrieve406(String message, java.lang.Object getGetallorganisationsthattheclientisauthorisedtoretrieve406) {
        super(message);
        this.getGetallorganisationsthattheclientisauthorisedtoretrieve406 = getGetallorganisationsthattheclientisauthorisedtoretrieve406;
    }

    public GETGetallorganisationsthattheclientisauthorisedtoretrieve406(String message, java.lang.Object getGetallorganisationsthattheclientisauthorisedtoretrieve406, java.lang.Throwable cause) {
        super(message, cause);
        this.getGetallorganisationsthattheclientisauthorisedtoretrieve406 = getGetallorganisationsthattheclientisauthorisedtoretrieve406;
    }

    public java.lang.Object getFaultInfo() {
        return this.getGetallorganisationsthattheclientisauthorisedtoretrieve406;
    }
}
